src/app/ViewPort/ViewPort.jsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * Viewport.jsx
+ *
+ * @description responsible for rendering interactive 3D nano car assemble screen
+ */
+
+import React, { useEffect, useRef } from "react";
+import * as THREE from "three";
+import { FPSMonitor } from "../commons";
+import { addLine, addMolecule } from "../../utils";
+import { nanocar } from "../../data";
+
+// get OrbitControls object
+const OrbitControls = require("three-orbit-controls")(THREE);
+
+/**
+ * Viewport
+ *
+ * @type {Function} renders Viewport component
+ */
+const Viewport = () => {
+  const viewportRef = useRef(null);
+
+  useEffect(() => {
+    // gets width of dom element
+    const width = viewportRef.current.clientWidth;
+    // gets height dom element
+    const height = viewportRef.current.clientHeight;
+
+    // three.js variables
+    const scene = new THREE.Scene();
+    const camera = new THREE.PerspectiveCamera(75, width / height, 0.1, 20000);
+    const light = new THREE.PointLight(0xffffff); // white light
+    const renderer = new THREE.WebGLRenderer({ antialias: true });
+    let frameId;
+
+    // OrbitControls
+    const controls = new OrbitControls(camera, renderer.domElement);
+
+    /**
+     * initOrbits
+     *
+     * @type {Function} initializes orbit parameters
+     *
+     */
+    const initOrbits = () => {
+      controls.rotateSpeed = 1.0;
+      controls.zoomSpeed = 1.2;
+      controls.panSpeed = 0.8;
+      controls.enableDamping = true;
+      controls.dampingFactor = 0.07;
+      controls.rotateSpeed = 0.07;
+      controls.update();
+    };
+
+    /**
+     * initCamera
+     *
+     * @type {Function} initializes the camera position
+     *
+     * @param {Number} x
+     * @param {Number} y
+     * @param {Number} z
+     *
+     */
+    const initCamera = (x = 0, y = 0, z = 0) => {
+      camera.position.x = x;
+      camera.position.y = y;
+      camera.position.z = z;
+    };
+
+    // axis parameters
+    const axisLength = 30;
+    const red = 0xe53935; // in respect to x-axis
+    const green = 0x00e676; // in respect to y-axis
+    const blue = 0x1976d2; // in respect to z-axis
+
+    // add axis lines
+    // x-axis
+    addLine(scene, [axisLength, 0, 0], [-axisLength, 0, 0], red);
```

Comment:
Can we move this axis helper to upper right of the screen?

Reply:
And add `x`, `y`, `z` labels if possible.

Reply:
My opinion this kind of changes would increase the PR complexity. I just reviewed the build process, warning, and file structure.  Let's create an issue for that. What do you think?

Reply:
That makes sense,I agree that this change is not in the scope of this PR. So opening a separate issue is a better idea.